### PR TITLE
Disable the multitrack option if the recording format is flv

### PR DESCRIPTION
**Pull request by RodrigoHolztrattner**
Disable the multitrack option when the selected recording format is set to `flv`.
This will also prevent saving a different multitrack bitmask if the user had it enabled on another format and decides to change to `flv`. It will force a bitmask that activates only the first audio track.

### CODE REVIEW

**Review by summeroff — APPROVED** [review]

File: tests/osn-tests/src/test_nodeobs_settings.ts [hunk 1]
```diff
@@ -1176,11 +1176,25 @@ describe('nodeobs_settings', function() {
...
+                }).parameters.find(parameter => {
+                    return parameter.name === 'RecFormat';
+                }).currentValue = selectedFormat;
+
                 osn.NodeObs.OBS_settings_saveSettings('Output', setCBR);
 
                 // Getting advanced output settings container with CBR parameters
                 let cbrOutputSettings = osn.NodeObs.OBS_settings_getSettings('Output');
-
+
```
Comment (EddyGharbi) [inline tests/osn-tests/src/test_nodeobs_settings.ts]: Could you clean the spaces here?

File: obs-studio-server/source/nodeobs_audio_encoders.cpp [hunk 2]
```diff
@@ -227,6 +227,15 @@ const char* GetAACEncoderForBitrate(int bitrate)
 
 #define INVALID_BITRATE 10000
 
+bool IsMultitrackAudioSupported(const char* format)
+{
+	if (format != nullptr && strcmp(format, "flv") == 0) {
```
Comment (EddyGharbi) [inline obs-studio-server/source/nodeobs_audio_encoders.cpp]: if the format is null we should be returning false right?

Comment (RodrigoHolztrattner) [reply obs-studio-server/source/nodeobs_audio_encoders.cpp]: Yes, fixed it

File: tests/osn-tests/src/test_nodeobs_settings.ts (hunk 1 above)
Comment (RodrigoHolztrattner) [reply tests/osn-tests/src/test_nodeobs_settings.ts]: Done

**Review by EddyGharbi — APPROVED** [review]
Looks good 👍

**Pull request merged**